oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -1242,6 +1242,62 @@ public String getReferenceCheckpoint() {
             return referenceCp;
         }
 
+        @Override
+        public String forceIndexLaneCatchup(String confirmMessage) throws CommitFailedException {
+
+            if (!"CONFIRM".equals(confirmMessage)) {
+                String msg = "Please confirm that you want to force the lane catch-up by passing 'CONFIRM' as argument";
+                log.warn(msg);
+                return msg;
+            }
+
+            if (!this.isFailing()) {
+                String msg = "The lane is not failing. This operation should only be performed if the lane is failing, it should first be allowed to catch up on its own.";
+                log.warn(msg);
+                return msg;
+            }
+
+            try {
+                log.info("Running a forced catch-up for indexing lane [{}]. ", name);
+                // First we need to abort and pause the running indexing task
+                this.abortAndPause();
```

Comment:
I wouldn't mind if this is just an additional preconditions... instead of doing it here, just verify it's done. But it's OK, no need to change it now.

(the idea is: We don't want this feature is used a lot, so making it a little bit harder to use might help that it is used less often... The "CONFIRM" is a great example of that!... Do not make it easy to delete stuff, so that it's less likely that stuff is deleted by mistake.)